core/gatewayclient/client.go

```diff
@@ -12,25 +12,25 @@ import (
 	"strings"
 
 	"github.com/kwilteam/kwil-db/core/client"
-	rpcClient "github.com/kwilteam/kwil-db/core/rpc/client"
+	rpcclient "github.com/kwilteam/kwil-db/core/rpc/client"
 	"github.com/kwilteam/kwil-db/core/rpc/client/gateway"
-	httpGateway "github.com/kwilteam/kwil-db/core/rpc/client/gateway/http"
-	httpTx "github.com/kwilteam/kwil-db/core/rpc/client/user/http"
+	gwClient "github.com/kwilteam/kwil-db/core/rpc/client/gateway/jsonrpc"
+	userClient "github.com/kwilteam/kwil-db/core/rpc/client/user/jsonrpc"
+	jsonrpc "github.com/kwilteam/kwil-db/core/rpc/json"
 	clientType "github.com/kwilteam/kwil-db/core/types/client"
-	gatewayTypes "github.com/kwilteam/kwil-db/core/types/gateway"
 )
 
 // GatewayClient is a client that is made to interact with a kwil gateway.
 // It inherits the functionality of the main Kwil client, but also provides
 // authentication cookies to the gateway.
 // It automatically handles the authentication process with the gateway.
 type GatewayClient struct {
-	client.Client
+	client.Client // core client
```

Comment:
GatwayClient still needs this type embeded, I cannot move it to `core/rpc/gateway/`, as `core/rpc` are just client for handling exact input/output of APIs, but `core/client`  deals with user input/outputs.